frameworks/Clojure/pedestal/install.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export JAVA_HOME=/usr/lib/jvm/java-1.7.0-openjdk-amd64
+
+export LEIN_HOME=$IROOT/lein
+
+fw_depends leiningen java
```

Comment:
I would add that he, as the first submitter since #1512 should update `leiningen` and have the install script 1) create `leiningen.installed`, 2) have that mimic the `java7.intalled` and export `LEIN_HOME` there, and 3) `source $IROOT/leiningen.installed` in his `setup.sh`